eng/liveBuilds.targets

```diff
@@ -0,0 +1,95 @@
+<Project>
+
+  <PropertyGroup>
+    <HostArch>$([System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture.ToString().ToLowerInvariant)</HostArch>
+    <CoreCLROSGroup Condition="'$(CoreCLROSGroup)' == ''">$(OSGroup)</CoreCLROSGroup>
```

Comment:
How are you dealing with OS subgroup?

Reply:
Currently, the local output directories in the CoreCLR product build don't include the OS subgroup IIRC, so I didn't need to do anything special here to handle it.

Reply:
Ahh, right you are, they don't.